# /controller/srv/SaveMap.srv
string caller # 调用者
---
bool success # 结果
